ome_zarr/axes.py

```diff
@@ -0,0 +1,100 @@
+"""Axes class for validating and transforming axes
+"""
+from typing import Any, Dict, List, Union
+
+from .format import Format
+
+KNOWN_AXES = {"x": "space", "y": "space", "z": "space", "c": "channel", "t": "time"}
+
+
+class Axes:
+    def __init__(self, axes: Union[List[str], List[Dict[str, str]]]) -> None:
+        self.axes = self._axes_to_dicts(axes)
+
+    def validate(self, fmt: Format) -> None:
+
+        # check names (only enforced for version 0.3)
+        if fmt.version == "0.3":
+            self._validate_axes_03()
+            return
+
+        self._validate_axes_types()
+
+    def get_axes(self, fmt: Format) -> Union[List[str], List[Dict[str, str]]]:
```

Comment:
Minor. `Axes.get_axes()` reads a bit redundant/confusing. If the output was a string, I would suggest to use Python built-in `__repr__` . I don't know if we have something that would clarify what this API does. `tolist()`?